testing/docker-master/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:xenial
+
+RUN apt update && apt dist-upgrade -y && apt install -y git && apt clean
+
+# enable multiverse as snapcraft cleanbuild does.
+RUN sed -i 's/ universe/ universe multiverse/' /etc/apt/sources.list
+
+# build and install snapcraft from master.
+git clone https://github.com/snapcore/snapcraft
+cd snapcraft
+apt build-dep -y ./
```

Comment:
That sounds great, but I don't know how to do it. Is there a way to get the list of packages that will be installed by apt ? I can't just remove all the dependencies, because some of them were already preinstalled in the machine.